assignments/calculator/solutions/Jlopezjlx/calc/calc.py

```diff
@@ -1,24 +1,40 @@
-print("CALCULADORA")
-print("OPCIONES: \n1-Suma\n2-Resta\n3-Multiplicar\n4-Dividir\n5-Restante")
+import maths
+loop = True
```

Comment:
You do not need this, check comment below.

Reply:
Fixed